tools/logs_agent_QA/README.md

```diff
@@ -0,0 +1,21 @@
+# Logs Agent Release QA
```

Comment:
It would be great to have some how-to's for common tasks here, too.  Something like:

 * I want to add a case to cover a potential regression
 * I want to add a new column (maybe including what the criteria are for a column -- if we start adding DSD tests, are those new columns?)
 * I want to add QA steps for a particular situation across platforms

Reply:
Good idea - ill add these :)